src/main/java/de/craften/ui/swingmaterial/MaterialPopupMenu

```diff
@@ -0,0 +1,46 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package de.craften.ui.swingmaterial;
+
+import de.craften.ui.swingmaterial.ElevationEffect;
+import de.craften.ui.swingmaterial.MaterialShadow;
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import javax.swing.JPopupMenu;
+import javax.swing.border.EmptyBorder;
+
+/**
+ *
+ * @author Rodrigo Butzke
+ */
+public class MaterialPopupMenu extends JPopupMenu {
+    private ElevationEffect elevation;
+    public MaterialPopupMenu() {
+        init();
+    }
+    public MaterialPopupMenu(String label) {
+        super(label);
+        init();
+    }
+    private void init() {
```

Comment:
I don't like this `init()` method. This should be done in the `MaterialPopupMenu()` constructor instead, while `MaterialPopupMenu(String label)` should call `this()` and `setLabel`.